appveyor.yml

```diff
@@ -38,8 +38,8 @@ build_script:
     - C:\msys64\usr\bin\bash -lc "pwd"
     - mkdir C:\projects\libiio\build-mingw-win32\"%configuration%"& cd C:\projects\libiio\build-mingw-win32\"%configuration%"
     - C:\msys64\usr\bin\bash -lc "pwd"
-    - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy pacman"
-    - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Su"
+    - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -U http://repo.msys2.org/msys/x86_64/pacman-5.2.1-6-x86_64.pkg.tar.xz"
+    - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Syu"
```

Comment:
Do you think this version will be used from now on? Or should add a 'To-Do' to check when latest pacman version will start working again and go back to that?

Reply:
I can add a TODO. Most probably the Appveyor will crash when it will update to the latest Msys version, so we will know when to update the script.